scripts/modules/b2b-account/quotes.js

```diff
@@ -37,17 +39,74 @@ define([
         render: function () {
             var self = this;
             Backbone.MozuView.prototype.render.apply(this, arguments);
-            var collection = new QuotesGridCollectionModel({ autoload: true });
+            var collection = new QuotesGridCollectionModel({ autoload: true });            
+            $('[data-mz-action="applyfilter"]').on('change input', function(e) {
+                e.preventDefault();
+                FILTERSTRING = "";
+                var dateValue ="";
```

Comment:
rename 'dateValue' to like 'expirationDateValue'. make more meningful

Reply:
done